### PR TITLE
add sprint page for scisprint 2023 April

### PR DESCRIPTION
**Pull request by terrychan999**

### CODE REVIEW

**Comment by yungyuc** [comment]
For #178

File: content/pages/sprint/2023/04-hsinchu.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+This training room will be like a workshop. There are some hands-on practices for you to get familiar with the course. Let's have fun with Mosky!
+
+For more information, you could check it out in `project list <https://hackmd.io/@sciwork/scisprint-2304#Projects>`__.
+If you have any questions, feel free to join `Discord <https://discord.gg/6MAkFrD>`__ and interact with us.
+
+Sign Up
+------------
+
+`Please register at kktix <#>`__ .
```
Comment (yungyuc) [inline content/pages/sprint/2023/04-hsinchu.rst:46]: Before the kktix page is made, this should be written "TBD" without a link.

File: content/pages/sprint/2023/04-hsinchu.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+:url: sprint/2023/04-hsinchu
+:save_as: sprint/2023/04-hsinchu.html
+
+Scisprint 2023 April in Hsinchu
+========================================
+
+* Date: 15th April, Saturday, 2023
+* Time: 10:00 -- 17:00 (7 hours)
+
+To join the sprint, please bring your laptop and `sign up at kktix <#>`__.  You are also very welcome to bring your project, which
```
Comment (yungyuc) [inline content/pages/sprint/2023/04-hsinchu.rst:15]: Before the kktix page is made, `sign up at kktix` should not be written.  It can be `sign up (TBD)`.

**Review by yungyuc — CHANGES_REQUESTED** [review]
We should publish the sciwork web page as soon as possible for promotion.  The kktix reg page can be made later.

File: content/pages/sprint/2023/04-hsinchu.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+Sign Up
+------------
+
+`Please register at kktix <#>`__ .
+
+Venue
+-----
+
+`前沿理論及計算研究中心 (國立清華大學第三綜合大樓 A 區 5 樓) <https://goo.gl/maps/EH2wWtkLQ8qLWd669>`__. Please see the `sprint
```
Comment (yungyuc) [inline content/pages/sprint/2023/04-hsinchu.rst:51]: Please also add the English name of the venue.

**Comment by terrychan999** [comment]
Ok, I got it.

**Review by yungyuc — APPROVED** [review]
LGTM

**Pull request merged**